.gitignore

```diff
@@ -57,5 +57,6 @@ ansible/inventories/mgmt-ocsit
 
 # Kitchen
 .kitchen
+Gemfile.lock
```

Comment:
I don't think we want to ignore this. It helps keep versions consistent between developers and CI (since we're only using ruby dependencies for kitchen).

Reply:
ok... restored